### PR TITLE
Change online lookup API to move identifier into body for security

### PR DESCRIPTION
**Pull request by ssrihari**
**Story card:** [ch4007](https://app.clubhouse.io/simpledotorg/story/4007/change-online-lookup-api-to-move-identifier-into-body-for-security)

### CODE REVIEW

File: app/schema/api/v4/models.rb [hunk 1]
```diff
@@ -365,6 +372,7 @@ def definitions
         lookup_patient: lookup_patient,
         lookup_patients: array_of("lookup_patient"),
         medical_history: Api::V3::Models.medical_history,
```
Comment (prabhanshuguptagit) [inline app/schema/api/v4/models.rb:374]: Is `medical_history` still being used?

File: spec/controllers/api/v4/patients_controller_spec.rb [hunk 2]
```diff
@@ -49,6 +49,17 @@ def set_headers(user, facility)
       expect(response_patient[:appointments].count).to eq 2
     end
 
+    it "works on patients without a medical history" do
```
Comment (prabhanshuguptagit) [inline spec/controllers/api/v4/patients_controller_spec.rb:52]:
```suggestion
    it "sends a nil medical history for patients without a medical history" do
```

**Review by prabhanshuguptagit — APPROVED** [review]
Looks good 👌🏼

File: app/schema/api/v4/models.rb (hunk 1 above)
Comment (ssrihari) [reply app/schema/api/v4/models.rb]: Good catch! 🙇  Fixed in 84964b8e.

**Pull request merged**